addons/mass_mailing/views/assets.xml

```diff
@@ -60,6 +60,14 @@
                     width: 100% !important;
                 }
             }
+            @media screen and (max-width: 1135px) {
+                td {
+                    max-width: inherit !important;
+                }
+                td > img:only-child {
+                    min-width: 100% !important;
+                }
+            }
```

Comment:
I won't be able to keep this in stable but I could introduce it in master and add in via js in stable. Either way I'm keeping it here for the moment so I don't forget.